internal/handler/completion.go

```diff
@@ -19,6 +19,7 @@ func (s *Server) handleTextDocumentCompletion(ctx context.Context, conn *jsonrpc
 	if err := json.Unmarshal(*req.Params, &params); err != nil {
 		return nil, err
 	}
+	params.LowercaseKeywords = s.getConfig().LowercaseKeywords
```

Comment:
I think it would be better to pass the config directly to the completer, because adding the completion option to the config would not require you to change it.